app/services/brewery_client.rb

```diff
@@ -0,0 +1,28 @@
+class BreweryClient
+  def self.find_brewery(name)
+    new(name).find_brewery(name)
+  end
+  # TODO: Ask Todd why I need this method above
```

Comment:
FYI this is a stylistic preference for how the method is evoked. I can understand some folks wanting it removed since it isnt necessary, and some folks prefer:

```ruby
Klass.perform(args)
```

over

```ruby
instance = Klass.new(args)
instance.perform
```

But you dont _need_ it. I also havent ever heard a convincing argument of why _not_ to do it, myself. Derrick may know more as he tends to prefer this pattern, and I am indifferent.